examples/run_ransac.py

```diff
@@ -0,0 +1,85 @@
+"""
+=================
+Run RANSAC
+=================
+
+
+In this example we show how to run the RANSAC of ``pyprep``.
+"""
+
+# Authors: Yorguin Mantilla <yjmantilla@gmail.com>
+#
+# License: MIT
+# Based On: use_noisy_module.py
+
+###############################################################################
+# First we import what we need for this example.
+import numpy as np
+import mne
+from scipy import signal as sx
+import time as clk
+
+from pyprep.find_noisy_channels import NoisyChannels
+
+###############################################################################
+# Now let's make some arbitrary MNE raw object for demonstration purposes.
+# We will think of good channels as sine waves and bad channels correlated with
+# each other as sawtooths. The idea is that the RANSAC is able to identify
+# these channels. We will need to set a montage because the RANSAC needs to
+# interpolate.
+
+sfreq = 1000.0
+
+montage = mne.channels.make_standard_montage("standard_1020")
```

Comment:
```suggestion
# We need a montage, because RANSAC uses spherical splines for interpolation
montage = mne.channels.make_standard_montage("standard_1020")
```